src/ProofOfConcepts/ConnectorTypesGallery/ConnectorTypesGallery.stories.tsx

```diff
@@ -33,21 +38,21 @@ const Template: ComponentStory<typeof ConnectorTypesGalleryWrapper> = (
 ) => <ConnectorTypesGalleryWrapper {...args} />;
 
 const categories = [
-  'AI/Machine learning',
-  'Serverless',
-  'Big Data',
-  'Database',
-  'Change data capture',
-  'Developer tools',
-  'Integration & delivery',
-  'Logging',
-  'Monitoring',
-  'CRM',
-  'Storage',
-  'Streaming & messaging',
-  'Amazon',
-  'Google',
-  'Azure',
+  'category-ai-machine-learning',
+  'category-serverless',
+  'category-big-data',
+  'category-database',
+  'category-change-data-capture',
+  'category-developer-tools',
+  'category-integration-and-delivery',
+  'category-logging',
+  'category-monitoring',
+  'category-crm',
+  'category-storage',
+  'category-streaming-and-messaging',
+  'category-amazon',
+  'category-google',
+  'category-azure',
 ];
```

Comment:
@dhirajsb how does this look for category label values?  These become keys to the entries in the i18n file.

Reply:
lgtm